android/src/main/java/com/dylanvann/fastimage/FastImageViewManager.java

```diff
@@ -156,10 +161,19 @@ public void setSrc(ImageViewWithUrl view, @Nullable ReadableMap source) {
                 .dontTransform()
                 .priority(priority)
                 .placeholder(TRANSPARENT_DRAWABLE)
-                .listener(LISTENER)
+                .listener(getListener())
                 .into(view);
     }
 
+    @ReactProp(name = "imageColor", customType = "Color")
```

Comment:
I think we want this to be `tintColor` so we don't have to change existing properties.

Reply:
the ios one is already named imageColor and i gave it that name on suggestion from TJ.. since we are using it out of our image wrapper class we only have to change the prop name in one place, but i could be convinced to change it

Reply:
Followed up in person, going to merge this as-is now and then change the name to tintColor in a separate pr